dig_test.go

```diff
@@ -1537,16 +1602,111 @@ func TestProvideInvalidGroup(t *testing.T) {
 	assert.Contains(t, err.Error(), `cannot parse group "foo,bar": invalid option "bar"`)
 }
 
-func TestProvideGroupAndName(t *testing.T) {
+func TestProvideInvalidAs(t *testing.T) {
+	ptrToStruct := &struct {
+		name string
+	}{
+		name: "example",
+	}
+	var nilInterface io.Reader
+	c := New()
```

Comment:
i would do container per subtest. there doesn't seem to be any need for sharing?